<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditCalc</class>
 <widget class="QWidget" name="CreditCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CREDIT CALCULATOR</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#CreditCalc {
	background-color: #1c1c1c;
}

QFrame#mainQFrame {
	border: 2px solid #66FCF1;
}

QRadioButton {
  color: #66FCF1;
}

QRadioButton::indicator {
    border: 1px solid #66FCF1;
    border-radius: 6px;
    width: 10px;
    height: 10px; 
}

QRadioButton::indicator:checked {
    background-color: #66FCF1;
}

QRadioButton::indicator:unchecked {
    background-color: #1c1c1c; 
}

QMessageBox {
    background-color: black;
}

QMessageBox QLabel {
    color: white;
}

QLabel {
	color: #66FCF1;
}

QPushButton {
	color: #66FCF1;
	border-radius: 1px;
	border: 1px solid #66FCF1;
	background-color: transparent;
}

QPushButton:pressed {
    color: #FFFFFF;
    background-color: #45A29E;
    border: 1px solid #45A29E;
}

QLineEdit {
	background-color: transparent;
	border-radius: 2px;
	border: 1px solid #66FCF1;
	color: #66FCF1;
}

QComboBox QAbstractItemView {
	background-color: #1c1c1c;
	selection-background-color: rgb(119, 118, 123);
	color: #66FCF1;
}

QComboBox {
	color: #66FCF1;
	border: 1px solid #66FCF1;
	background-color: transparent;
}</string>
  </property>
  <widget class="QFrame" name="mainQFrame">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>3</y>
     <width>348</width>
     <height>410</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>-11</y>
      <width>325</width>
      <height>285</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="inputQGridLayout">
     <item row="2" column="2">
      <widget class="QLabel" name="procentQLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>%</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="amountQLineEdit">
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="amountOfCreditQLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Amount of credit</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="creditTermQLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Credit term</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="rubQLabel_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>RUB</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="termQLineEdit">
       <property name="maxLength">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QVBoxLayout" name="typeOfPaymentQVBoxLayout">
       <item>
        <widget class="QRadioButton" name="annuityQRadioButton">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Annuity</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="differentialQRadioButton">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Differential</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="rateQLineEdit">
       <property name="maxLength">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="typeOfPaymentsQLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Type of payments</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="calculateQPushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>CALCULATE</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="dateQComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <italic>false</italic>
        </font>
       </property>
       <item>
        <property name="text">
         <string>YEARS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MONTH</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="interestRateQLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Interest rate</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>272</y>
      <width>330</width>
      <height>128</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="outputQGridLayout">
     <item row="1" column="2">
      <widget class="QLabel" name="rubQLabel">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>37</height>
        </size>
       </property>
       <property name="text">
        <string>RUB</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="totalPaymentQLabel">
       <property name="text">
        <string>Total payment</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="totalPaymentOutputQLabel">
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="rubQLabel_2">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>37</height>
        </size>
       </property>
       <property name="text">
        <string>RUB</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="overpaymentQLabel">
       <property name="text">
        <string>Overpayment</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="monthlyPaymentQLabel">
       <property name="text">
        <string>Monthly payment</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="rubQLabel_3">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>37</height>
        </size>
       </property>
       <property name="text">
        <string>RUB</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="overpaymentOutputQLabel">
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="monthlyPaymentOutputQLabel">
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="Line" name="horizontalQLine">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>265</y>
     <width>347</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame[frameShape=&quot;4&quot;],
QFrame[frameShape=&quot;5&quot;]
{
    border: none;
    background: #66FCF1;
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
